frontends/p4/substitution.h

```diff
@@ -76,21 +76,11 @@ class TypeVariableSubstitution final : public TypeSubstitution<const IR::ITypeVa
  public:
     TypeVariableSubstitution() = default;
     TypeVariableSubstitution(const TypeVariableSubstitution& other) = default;
-    /* Set bindings for multiple variables at the same time.
-     * @param params   List of variables to set.
-     * @param args     List of type arguments to set to variables.
-     * @param root     P4 program element which is being processed. Used for error reporting.
-     * @return         True on success. */
-    bool setBindings(const IR::Node* root,
```

Comment:
is there any particular reason for deleting the comments rather than updating them?